src/Files.App/Actions/FileSystem/OpenItemAction.cs

```diff
@@ -70,7 +73,7 @@ public OpenItemWithApplicationPickerAction()
 
 		public async Task ExecuteAsync()
 		{
-			NavigationHelpers.OpenSelectedItems(context.ShellPage, true);
+			await NavigationHelpers.OpenSelectedItems(context.ShellPage, true);
 		}
```

Comment:
You can remote `async` from the declaration and then return `NavigationsHelpers.Open...`

Reply:
Done